### PR TITLE
add class-rebalancing doc

### PR DESCRIPTION
**Pull request by JonathanGuerne**

### CODE REVIEW

**Comment by JonathanGuerne** [comment]
I noticed that the staging url (https://class-rebalancing-swiss-ai-center.kube.isc.heia-fr.ch) seems to link to an unrelated page (about a pet clinic), if that is a mistake on my part let me know

**Comment by andrptrc** [comment]
> I noticed that the staging url (https://class-rebalancing-swiss-ai-center.kube.isc.heia-fr.ch) seems to link to an unrelated page (about a pet clinic), if that is a mistake on my part let me know

Try to run the Workflow again if you check the last run in the Action section you can see that the service has not been deployed.

**Comment by rmarquis** [comment]
The deployment should happen only for documentation in the main branch (as per the workflow deploy condition).
I feel there's an hidden story about that pet clinic, but I'm afraid to ask haha.

File: docs/reference/index.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ mdwrap is a command line tool to format our Markdown documentation made with
 | ---------------------------------------------------------------------- | ----------------------------------------------------------------------- | ------------------------------------------------------------------------ | ------------------------------------------------------------- |
 | [ae-ano-detection](./services/ae-ano-detection.md)                     | <https://github.com/swiss-ai-center/ae-ano-detection-service>           | <https://ae-ano-detection-swiss-ai-center.kube.isc.heia-fr.ch>           | <https://ae-ano-detection-service.swiss-ai-center.ch>         |
 | [average-shade](./services/average-shade.md)                           | <https://github.com/swiss-ai-center/average-shade-service>              | <https://average-shade-swiss-ai-center.kube.isc.heia-fr.ch>              | <https://average-shade-service.swiss-ai-center.ch>            |
+| [class-rebalancing](./services/class-rebalancing.md)     | <https://github.com/swiss-ai-center/class-rebalancing-service>   | <https://class-rebalancing-swiss-ai-center.kube.isc.heia-fr.ch>   | <https://class-rebalancing-service.swiss-ai-center.ch> |
```
Comment (rmarquis) [inline docs/reference/index.md:40]: Nitpicking here, but can we have proper spacing to get a nicer alignment? That might ease future review of that table.

**Review by rmarquis — CHANGES_REQUESTED** [review]

File: docs/reference/index.md (hunk 1 above)
Comment (rmarquis) [reply docs/reference/index.md]: Nvm, I fixed it and cleaned the rest of the table too.

**Review by rmarquis — APPROVED** [review]

**Comment by rmarquis** [comment]
@JonathanGuerne For the service deploiement, maybe check you correctly set the environment variables in settings (I don't seem to have access to the settings of that repo).

**Comment by JonathanGuerne** [comment]
@rmarquis I followed your advice and made sure the env variable were correctly set. (also you should have access now)

The dev deployment is working (I tested the service already), only the api-doc url mention above still redirect to the wrong ressource for now

**Comment by rmarquis** [comment]
This looks like an issue on the dev cluster to me. As far as I remember, there is no explicit special configuration needed for the docs rediction URL. The docs is correctly [available online](https://class-rebalancing-swiss-ai-center.kube.isc.heia-fr.ch/docs), only the redirect is not configured properly.

I've deployed a service just a 2 weeks ago, without any issue. I have no explanation here.
Maybe try to deploy in production, and check that the docs redirection behaves as expected? If it does, this might be a sign of a recent misconfiguration on the dev cluster.

**Comment by andrptrc** [comment]
> @rmarquis I followed your advice and made sure the env variable were correctly set. (also you should have access now)
> 
> The dev deployment is working (I tested the service already), only the api-doc url mention above still redirect to the wrong ressource for now

Which link exactly is not working?

**Comment by JonathanGuerne** [comment]
This one: https://class-rebalancing-swiss-ai-center.kube.isc.heia-fr.ch/

**Comment by andrptrc** [comment]
> This one: https://class-rebalancing-swiss-ai-center.kube.isc.heia-fr.ch/

It seems to work for me, It shows the Swagger API, what does it show to you ?

**Comment by JonathanGuerne** [comment]
It redirects me to this url: https://class-rebalancing-swiss-ai-center.kube.isc.heia-fr.ch/petclinic/ (a landing page for a pet clinic apparently)

**Comment by andrptrc** [comment]
> It redirects me to this url: https://class-rebalancing-swiss-ai-center.kube.isc.heia-fr.ch/petclinic/ (a landing page for a pet clinic apparently)

Did you try to clear your cache and cookies ? This page is the default page for nginx I guess and for me it redirects correctly

**Comment by JonathanGuerne** [comment]
You're right, it worked ^^'
thanks and sorry for that

**Comment by andrptrc** [comment]
No problem glad to help 😉

**Pull request merged**